introduction.md

```diff
@@ -1,2 +1,7 @@
 It's me, Mario! Ooops, sorry...It's me, Lucas!
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?